File: c++/CFSQRT.cc
#include <iostream>
#include <math.h>
using namespace std;

int main()
{
	int z;
	cin >> z;
	while(z>0)
	{
		int a;
		cin >> a;
		cout << int(sqrt(a)) << endl;
		z-=1;
	}
}

File: java/CFSQRT.java
import java.util.Scanner;

public class CFSQRT {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int z = sc.nextInt();
        while (z>0){
            int a = sc.nextInt();
            int result = (int) Math.sqrt(a);
            System.out.println(result);
            z--;
        }
    }
}
